pylot/perception/flags.py

```diff
@@ -84,6 +84,16 @@
                   ['kitti_tracking', 'coco', 'mot', 'nuscenes'],
                   'CenterTrack available models')
 
+# QDTrack tracking flags.
+flags.DEFINE_string(
+    'qd_track_model_path',
+    'dependencies/models/tracking/qd_track/qdtrack-frcnn_r50_fpn_12e_bdd100k-13328aed.pth', #FIX
+    'Path to the model')
+flags.DEFINE_string(
+    'qd_track_config_path',
+    'dependencies/qdtrack/configs/qdtrack-frcnn_r50_fpn_12e_bdd100k.py', #FIX
+    'Path to the model')
```

Comment:
I guess this is the Path to the model configuration.

Reply:
Yea, it's the qd track model path but it's currently the path to the bdd model, rather than the waymo model